calico-enterprise/networking/configuring/node-local-dns-cache.mdx

```diff
@@ -0,0 +1,227 @@
+---
+description: Configure Calico Enterprise for NodeLocal DNSCache
+---
+
+# Configure Calico Enterprise for NodeLocal DNSCache
+
+## Big picture
+
+Configure Calico Enterprise for clusters with NodeLocal DNSCache feature provided by Kubernetes on versions 1.18+. 
```

Comment:
{{prodname}} throughout (except in the title).

Reply:
done